src/data/move.ts

```diff
@@ -1804,7 +1804,7 @@ export class StealHeldItemChanceAttr extends MoveEffectAttr {
         const stolenItem = tierHeldItems[user.randSeedInt(tierHeldItems.length)];
         user.scene.tryTransferHeldItemModifier(stolenItem, user, false).then(success => {
           if (success) {
-            user.scene.queueMessage(getPokemonMessage(user, ` stole\n${target.name}'s ${stolenItem.type.name}!`));
+            user.scene.queueMessage(i18next.t("moveTriggers:stoleItem", {pokemonName: getPokemonNameWithAffix(user), targetName: target.name, ItemName: stolenItem.type.name}));
```

Comment:
```suggestion
            user.scene.queueMessage(i18next.t("moveTriggers:stoleItem", {pokemonName: getPokemonNameWithAffix(user), targetName: getPokemonNameWithAffix(target), ItemName: stolenItem.type.name}));
```

The same for every other please

Reply:
But in the original code, the targetName parameter uses target.name instead of getPokemonNameWithAffix(target). Sure it should be modified like this?"

Reply:
> But in the original code, {{targetName}} uses target.name instead of getPokemonNameWithAffix(target). Sure it should be modified like this?

Yes. The old code was just wrong. Please make the changes i suggested. If not sure ask Temp or Walker

Reply:
fixed